forloop_common_structures/core/job.py

```diff
@@ -63,7 +45,7 @@ class PipelineElements(TypedDict):
 @dataclass
 class PipelineJob:
     pipeline_uid: str
-    pipeline_elements: PipelineElements
+    pipeline_elements: PipelineElements  # TODO: Discuss and remove (?) when PrototypeJobs are implemented - current pipeline on the backend should be used
```

Comment:
please let's name it "pipeline_building_blocks" - elements is used for too many other things in the platform